tests/bson/bson-toPHP-003.phpt

```diff
@@ -0,0 +1,470 @@
+--TEST--
+BSON\toPHP(): Tests from serialization specification
+--SKIPIF--
```

Comment:
It should have the same test cases (from the spec itself), but it wasn't using traits or short array syntax. Additionally, I'm using the dynamic BSON namespace and encode/decode functions that @bjori created in `tools.php`.